server/views/applications/pages/basic-information/placement-purpose.njk

```diff
@@ -14,6 +14,7 @@
       id: "otherReason",
       name: "otherReason",
       type: "text",
+      errorMessage: errors.placementPurposes,
```

Comment:
Should this error be `errors.otherReason`?

Reply:
Yes

Reply:
It's made me think about adding a macro for these form fields, because they're VERY repetitive, and it's too easy to miss stuff

Reply:
Yeah, a wrapper around the GDS macros wouldn't be a bad idea

Reply:
Having a look at this now 👍